dev/generate-kernel-signatures.py

```diff
@@ -12,61 +12,61 @@
+    # "awkward_NumpyArray_getitem_next_at",
+    # "awkward_RegularArray_getitem_next_array_advanced",
+    # "awkward_ByteMaskedArray_toIndexedOptionArray",
+    # "awkward_combinations",  # ?
+    # "awkward_IndexedArray_simplify",
     "awkward_ListArray_validity",
-    "awkward_UnionArray_validity",
-    "awkward_index_carry",
-    "awkward_ByteMaskedArray_getitem_carry",
-    "awkward_IndexedArray_validity",
-    "awkward_ByteMaskedArray_overlay_mask",
-    "awkward_NumpyArray_reduce_mask_ByteMaskedArray_64",
-    "awkward_RegularArray_getitem_carry",
-    "awkward_NumpyArray_getitem_next_array",
-    "awkward_RegularArray_localindex",
-    "awkward_NumpyArray_contiguous_next",
-    "awkward_NumpyArray_getitem_next_range",
-    "awkward_NumpyArray_getitem_next_range_advanced",
-    "awkward_RegularArray_getitem_next_range",
-    "awkward_RegularArray_getitem_next_range_spreadadvanced",
-    "awkward_RegularArray_getitem_next_array",
-    "awkward_missing_repeat",
-    "awkward_Identities_getitem_carry",
-    "awkward_RegularArray_getitem_jagged_expand",
-    "awkward_ListArray_getitem_jagged_expand",
-    "awkward_ListArray_getitem_next_array",
-    "awkward_RegularArray_broadcast_tooffsets",
-    "awkward_NumpyArray_fill_tobool",
-    "awkward_NumpyArray_reduce_adjust_starts_64",
-    "awkward_NumpyArray_reduce_adjust_starts_shifts_64",
-    "awkward_regularize_arrayslice",
-    "awkward_RegularArray_getitem_next_at",
-    # "awkward_ListOffsetArray_compact_offsets", Need to tune tests
+    # "awkward_UnionArray_validity",
+    # "awkward_index_carry",
+    # "awkward_ByteMaskedArray_getitem_carry",
+    # "awkward_IndexedArray_validity",
+    # "awkward_ByteMaskedArray_overlay_mask",
+    # "awkward_NumpyArray_reduce_mask_ByteMaskedArray_64",
+    # "awkward_RegularArray_getitem_carry",
+    # "awkward_NumpyArray_getitem_next_array",
+    # "awkward_RegularArray_localindex",
+    # "awkward_NumpyArray_contiguous_next",
+    # "awkward_NumpyArray_getitem_next_range",
+    # "awkward_NumpyArray_getitem_next_range_advanced",
+    # "awkward_RegularArray_getitem_next_range",
+    # "awkward_RegularArray_getitem_next_range_spreadadvanced",
+    # "awkward_RegularArray_getitem_next_array",
+    # "awkward_missing_repeat",
+    # "awkward_Identities_getitem_carry",
+    # "awkward_RegularArray_getitem_jagged_expand",
+    # "awkward_ListArray_getitem_jagged_expand",
+    # "awkward_ListArray_getitem_next_array",
+    # "awkward_RegularArray_broadcast_tooffsets",
+    # "awkward_NumpyArray_fill_tobool",
+    # "awkward_NumpyArray_reduce_adjust_starts_64",
+    # "awkward_NumpyArray_reduce_adjust_starts_shifts_64",
+    # "awkward_regularize_arrayslice",
+    # "awkward_RegularArray_getitem_next_at",
+    # # "awkward_ListOffsetArray_compact_offsets", Need to tune tests
     "awkward_BitMaskedArray_to_ByteMaskedArray",
-    "awkward_BitMaskedArray_to_IndexedOptionArray",
+    # "awkward_BitMaskedArray_to_IndexedOptionArray",
```

Comment:
This is the end of a long commented-out block. For git-related reasons, it's better to delete than comment-out.

I still use commenting-out a lot in short-term work and in the midst of a PR that isn't ready to be reviewed, but when these things get committed to history, let's do the simple thing and delete them outright.